test/datalog_tests/lib_test.debug.ast.expected

```diff
@@ -1360,7 +1374,7 @@ json_test::JsonTest[(json_test::JsonTest{.description=json_test::scalar2(), .val
 json_test::JsonTest[(json_test::JsonTest{.description=json_test::scalar3(), .value=json_test::to_json_string_or_default((json::from_json_string(json_test::scalar3()): std::Result<json::JsonValue,string>))}: json_test::JsonTest)].
 json_test::JsonTest[(json_test::JsonTest{.description=json_test::scalar4(), .value=json_test::to_json_string_or_default((json::from_json_string(json_test::scalar4()): std::Result<json::JsonValue,string>))}: json_test::JsonTest)].
 json_test::JsonTest[(json_test::JsonTest{.description=json_test::scalar5(), .value=json_test::to_json_string_or_default((json::from_json_string(json_test::scalar5()): std::Result<json::JsonValue,string>))}: json_test::JsonTest)].
-json_test::ODeserialized[x] :- json_test::Deserialized[(__json_test_deserialized0@ (x: json_test::TaggedEnum))], Inspect debug::debug_event((32'd163, 32'd0, 32'd0), ddlog_weight, ddlog_timestamp, "Map", __json_test_deserialized0, x).
+json_test::ODeserialized[x] :- json_test::Deserialized[(__json_test_deserialized0@ (x: json_test::TaggedEnum))], Inspect debug::debug_event((32'd165, 32'd0, 32'd0), ddlog_weight, ddlog_timestamp, "Map", __json_test_deserialized0, x).
```

Comment:
would be nice to get rid of these spurious diffs, but I don't have a concrete proposal.